src/js/js-loader.ts

```diff
@@ -33,9 +33,9 @@ function jsLoader(source: string, sourceMaps: any): void {
 }
 
 function extractTranslations(loader: TranslateLoaderContext, source: string, sourceMaps: any): void {
-    const options = loaderUtils.parseQuery(loader.query);
+    const options = (loaderUtils as any).getOptions(loader) || {};
```

Comment:
I don't think query is a string in webpack v2.

it can also be an object: [docs](https://github.com/webpack/loader-utils#getoptions)